ui/src/main/webapp/src/app/shared/validators/project-name-not-exists.validator.ts

```diff
@@ -25,7 +29,10 @@ export class ProjectNameNotExistsValidator implements Validator {
                 if (result == null) {
                     resolve(null);
                 } else {
-                    resolve({['nameIsTaken']: true});
+                    if (this.project != null && result == this.project.id)
```

Comment:
Yeah, I guess we could not send it if the user didn't change anything at all. :) But that gets a little tricky as the user is presently updating the name on the same object. We'd have to pass in the original title and id instead, I think. Maybe something for the future.